### PR TITLE
Separate collaborators from contributors files

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

* Lists collaborations in this page, separated from the contributors file

### List of changes proposed in this PR (pull-request)

* Ask Ale to add her bio (@AlexandraAAJ)
* Add Chris' + SciLifeLab details (@libcce)
* Move all collab info to a separate file (@Karvovskaya to add VU Amsterdam info and her as a rep)
* Check with core delivery team who is missing

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
@da5nsy Can you help troublesheet why the contributors.md data is not being displayed with the most recent version? Would I need to manually build the book locally or if the workflow takes care of that?

**Review by libcce — APPROVED** [review]

**Comment by EKaroune** [comment]
I fixed the CI issue in a PR yesterday so hopefully you can now merge this issue.

**Comment by kallewesterling** [comment]
@malvikasharan Testing the CI on my end, and I believe these are the culprits:

- @AlexandraAAJ's ORCID is an empty reference in the `contributors.md` file (line 93)
- The link to @bsipocz's GitHub profile is incorrect (line 445)
- The link to @bsipocz's ORCID profile is incorrect (line 446)

I will try to fix and push something for these.

**Comment by kallewesterling** [comment]
After chatting with @AlexandraAAJ - I've added in her bio here as well.

@malvikasharan - let me know if I can help resolve anything else here :)

File: book/website/afterword/collaborators.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+### Representative
+- [Christopher Erdmann](https://the-turing-way.netlify.app/afterword/contributors-record#christopher-erdmann), Head of Open Science, SciLifeLab Data Centre
+
+(collaborators-eds)=
+## [Environmental Data Science Book](https://edsbook.org/welcome.html)
+
+Environmental Data Science book or EDS book is a living, open and community-driven online resource to showcase and support the publication of data, research and open-source tools for collaborative, reproducible and transparent Environmental Data Science.
+
+### Representative
+- [Alejandro Coca](https://the-turing-way.netlify.app/afterword/contributors-record#alejandro-coca), Senior Researcher, Environment and Sustainability Research Programme, The Alan Turing Institute
```
Comment (aleesteele) [inline book/website/afterword/collaborators.md:59]: Adding @acocac's name here!
```suggestion
- [Alejandro Coca-Castro](https://the-turing-way.netlify.app/afterword/contributors-record#alejandro-coca), Senior Researcher, Environment and Sustainability Research Programme, The Alan Turing Institute
```

File: book/website/afterword/collaborators.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+(collaborators)=
+# Collaborating Organisations and Projects
+
+*The Turing Way works in collaboration and formal partnership with many communities and organisations.
```
Comment (aleesteele) [inline book/website/afterword/collaborators.md:4]:
```suggestion
_The Turing Way_ works in collaboration and formal partnership with many communities and organisations.
```

File: book/website/afterword/collaborators.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+The project also receives in-kind support through contributions from members who represent their communities, projects or organisations in The Turing way.
```
Comment (aleesteele) [inline book/website/afterword/collaborators.md:5]:
```suggestion
The project also receives in-kind support through contributions from members who represent their communities, projects or organisations in The Turing way.
These contributions may take different forms, including the sharing of practices, documentation, presentations, Scriberia images, and other resources.
```

**Review by aleesteele — APPROVED** [review]

**Comment by aleesteele** [comment]
Added a few small notes @malvikasharan! Hopefully that helps.

File: contributors.md [hunk 4]
```diff
@@ -99,32 +98,54 @@ I am a Mozilla fellowship (2016) and Fulbright scholarship (2007) alumna.
 > Prior to that, she lived in Peru where she was the CEO of Teatro La Plaza, one of Latin America’s leading theatre companies. There, she oversaw the production of over 100 theatre plays for adults and children, such as Mamma Mia!, Billy Elliot, The curious incident of the Dog in the Nighttime, and Hamlet (with a cast of actors with Down syndrome), among many others.
```
Comment (AlexandraAAJ) [inline contributors.md:98]:
```suggestion
> Prior to that, I lived in Peru where she was the CEO of Teatro La Plaza, one of Latin America’s leading theatre companies. There, she oversaw the production of over 100 theatre plays for adults and children, such as Mamma Mia!, Billy Elliot, The curious incident of the Dog in the Nighttime, and Hamlet (with a cast of actors with Down syndrome), among many others.
```

Comment (AlexandraAAJ) [inline contributors.md:98]:
```suggestion
> Prior to that, I lived in Peru where she was the CEO of Teatro La Plaza, one of Latin America’s leading theatre companies. There, she oversaw the production of over 100 theatre plays for adults and children, such as Mamma Mia!, Billy Elliot, The curious incident of the Dog in the Nighttime, and Hamlet (with a cast of actors with Down syndrome), among many others.
```

**Comment by malvikasharan** [comment]
Tagging @Arielle-Bennett for reviewing and updating the RIR informal initiative section here: https://github.com/the-turing-way/the-turing-way/pull/3334/files#diff-f2c642ac7acf3965672e82416816e29b864abb0f4b9cc36f66b08505616d23c1

File: book/website/afterword/subprojects.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+- [Malvika Sharan](https://the-turing-way.netlify.app/afterword/contributors-record#malvika-sharan)
+- [Kirstie Whitaker](https://the-turing-way.netlify.app/afterword/contributors-record#kirstie-whitaker)
+- [Alexandra Araujo Alvarez](https://the-turing-way.netlify.app/afterword/contributors-record#alexandra-araujo-alvarez)
+
+### Pathways Python Package for Curated Access to Book Chapters
+
+*Project Information: [GitHub repo in The Turing Way](https://github.com/the-turing-way/pathways) and [prototype repo in the Turing organisation](https://github.com/alan-turing-institute/bio-Turing-Way/).*
+
+**Developers**
+- May Yong, Iain Stanso, Lydia France, Malvika Sharan: funded by AI for Science and Government in 2021-2022
```
Comment (malvikasharan) [inline book/website/afterword/subprojects.md:64]:
```suggestion
- May Yong, Iain Stanson, Lydia France, Malvika Sharan: funded by AI for Science and Government in 2021-2022
```

File: book/website/afterword/collaborators.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
+(collaborators)=
+# Collaborating Organisations and Projects
+
+_The Turing Way_ works in collaboration and formal partnership with many communities and organisations.
+The project also receives in-kind support through contributions from members who represent their communities, projects or organisations in The Turing way.
+These contributions may take different forms, such as sharing of practices, co-creating documentation, designing workshops and presentations, co-hosting events, creating illustrations, providing mentorship, leading or contributing to sub-projects and co-developing other resources.
+Some of these contributions also include projects that build alongside or upon The Turing Way resources or collaborate with The Turing Way team members in various capacities.
+We acknowledge each of these contributing members individually and list their profiles under “Collaborating organisations and projects”.*
```
Comment (malvikasharan) [inline book/website/afterword/collaborators.md:8]:
```suggestion
We acknowledge each of these contributing members individually and list their profiles under “Collaborating organisations and projects”.
```

File: book/website/afterword/subprojects.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+# Working Groups, Funded Projects and Informal Initiatives
+
+This page provides access to various group-based initiatives or subprojects, listed under three themes: working groups that guide the different areas of work as a formal part of Turing's governance, funded projects that are led by members in a paid capacity and informal initiatives that are led by members through collaboration around shared interests.
+These subprojects are often time-bound and are supported by community members who take open leadership roles in response to the emergent and timely opportunities in the research and open science ecosystem.
+
+## Working Groups Led by Community Members
+
+### Translation and Localisation Working Group Leads
+
+*Project Information: [Chapter in the Community Handbook](https://the-turing-way.netlify.app/community-handbook/translation)**
```
Comment (malvikasharan) [inline book/website/afterword/subprojects.md:11]:
```suggestion
*Project Information: [Chapter in the Community Handbook](https://the-turing-way.netlify.app/community-handbook/translation)*
```

**Comment by malvikasharan** [comment]
Any leftover information should be added with a new PR, an issue has been created to capture: #3442 3442

File: contributors.md [hunk 8]
```diff
@@ -286,73 +341,98 @@ I am a Mozilla fellowship (2016) and Fulbright scholarship (2007) alumna.
...
+> I am an astronomer turned Research Software Engineer. I work at Caltech/IPAC to build and improve tools, e.g. Python libraries and Science Platforms to provide ways to access data in the NASA/IPAC Infrared Science Archive. Prior joining IPAC I was DiRAC Fellow in the data engineering team at the Institute for Data Intensive Research in Astrophysics and Cosmology in Seattle. I am a developer and maintainer of several open source astronomy libraries and their infrastructure (e.g. astroML, astroquery, astropy) and I enjoy contributing to upstream projects as well. I have a keen interest in finding ways to promote the open development model and make tools more sustainable. I am a fellow of the Software Sustainability Institute.
```
Comment (bsipocz) [inline contributors.md:432]: I'm going through my GH mentions from the past few weeks and noticed that some of this file has shifted around. E.g. here Benjamin's and mine are twined around (both twitter and personal highlights). I haven't checked, but there maybe other instances of the same problem, too.

**Pull request merged**